notes-to-self/loopy.py

```diff
@@ -6,7 +6,7 @@
 async def loopy():
     try:
         while True:
-            time.sleep(0.01)
+            time.sleep(0.01)  # noqa: ASYNC101  # OS sleep so no CPU fire
```

Comment:
I feel like the comment can be reworded slightly :^)

Reply:
Re-worded comment in 1b9643d2